gradboost_pv/models/basic.py

```diff
@@ -6,30 +6,23 @@
 
 from gradboost_pv.models.utils import (
     TRIG_DATETIME_FEATURE_NAMES,
-    ORDERED_NWP_FEATURE_VARIABLES,
     build_lagged_features,
 )
 
 
-def load_local_preprocessed_slice(forecast_horizon_step: int) -> np.ndarray:
-    return np.load(
-        f"/home/tom/local_data/basic_processed_nwp_data_step_{forecast_horizon_step}.npy"
+def load_local_preprocessed_slice(forecast_horizon_step: int) -> pd.DataFrame:
+    return pd.read_pickle(
+        f"/home/tom/local_data/basic_processed_nwp_data_step_{forecast_horizon_step}.p"
```

Comment:
perhaps worth adding a #TODO here, so we dont forget about the local file

Reply:
should the name of the file be `.pickle`, rather than `.p` to make it very clear what it is

Reply:
Sure can do